draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -2761,6 +2792,9 @@ Suite ID fields MUST be chosen as follows:
     2. For expand\_message\_xmd ({{hashtofield-expand-xmd}}) with SHA3-256,
        HASH\_ID is "XMD:SHA3-256".
 
+Suites that use an alternative hash\_to\_field function that meets the requirements
+in {{hashtofield-exteff}} MUST indicate this by choosing a unique identifier for HASH\_ID.
```

Comment:
```suggestion
  Suites that use an alternative hash\_to\_field function that meets the requirements
  in {{hashtofield-exteff}} MUST indicate this by appending a tag identifying that function
  to the HASH\_ID field, separated by a colon (":", ASCII 0x3A).
```

Two suggestions:

(1) add indentation to keep this as part of the bulleted list, and

(2) "choosing a unique identifier" is not quite explicit enough, because it's not clear what it needs to be different from.

Reply:
> 👍 this looks great, with one small comment. What do you think of the suggested edit?

This works for me!